README.md

```diff
@@ -122,13 +122,23 @@ Compiling requires using CMake 3.1 or newer. GCC and Clang are both known to wor
 
 #### Docker building
 
-The recommended way to build for most platforms is to use Docker. Several Docker images are provided that contain the requisite toolchain and dependencies for building mGBA across several platforms.
+The recommended way to build for most platforms is to use Docker. Several Docker images are provided that contain the requisite toolchain and dependencies for building mGBA across several platforms. 
+
+Note: If you are on Windows, you need to make sure your terminal session is inside the mGBA repository root directory before using Docker. (See issue [#1985](https://mgba.io/i/1985) for details.) If you are on an older Windows system, you may need to configure your Docker to use VirtualBox shared folders to correctly map your current `mgba` checkout directory to the Docker image's working directory.
+
+For example, on Windows, after cloning the repository, we will assume the `mgba` checkout directory is located at `X:\github\mgba`. What you want to do is to open up the terminal (Powershell or WSL) and navigate to the `mgba` root directory before proceeding:
+
+	cd X:\github\mgba
 
 To use a Docker image to build mGBA, simply run the following command while in the root of an mGBA checkout:
 
-	docker run --rm -t -v ${PWD}:/home/mgba/src mgba/windows:w32
+	docker run --rm -it -v ${PWD}:/home/mgba/src mgba/windows:w32
 
-This will produce a `build-win32` directory with the build products. Replace `mgba/windows:w32` with another Docker image for other platforms, which will produce a corresponding other directory. The following Docker images available on Docker Hub:
+Depending on which terminal you are running the command from and whether your current working path may contain spaces or special characters, you may need to change the `${PWD}` to `$PWD` or `"$PWD"` to run the command.
```

Comment:
I don't think it's up to mGBA to explain how your shell works.

Reply:
Removing it. I thought it might be helpful to explain a bit about the curly brackets, and why on different platforms and terminal types, you need to remove curly brackets. I thought that, without this explanation, people would just straight up copy/paste the command without modifying it.

Reply:
To some extent you're right, but also this section won't stop them from copy/pasting it. Putting the most widely compatible version there is probably a good idea. I think that might be `"${PWD}"`.